lib/src/specs/icon/icon_widget.dart

```diff
@@ -1,11 +1,6 @@
 import 'package:flutter/material.dart';
 
-import '../../core/styled_widget.dart';
-import '../../decorators/widget_decorator_widget.dart';
-import '../../factory/mix_provider.dart';
-import '../../factory/mix_provider_data.dart';
-import 'icon_attribute.dart';
-import 'icon_spec.dart';
+import '../../../mix.dart';
```

Comment:
Just be aware of this, vscode does this all the time to consolidate the exports. I wonder if there is a way to make sure it does not use mix.dart as an import